src/Prefabs/Resources/Mining/IronNugget.et

```diff
@@ -41,7 +40,7 @@ GenericEntity {
       ItemRenderRotation 0 0 90
      }
     }
-    m_SlotType SLOT_LOOT_STORAGE
+    m_Size SLOT_2x1
```

Comment:
```suggestion
    m_Size SLOT_1x1
```

Reply:
If 1x1 CameraDistanceToItem: 0.275

Reply:
Yeah good idea to make them 1x1, I think that is better. I adjusted their rotation and scale to be this:
![image](https://user-images.githubusercontent.com/8494013/188406471-45e49df2-fd66-4587-8bca-dbc2f2b9fda8.png)